.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  env: {
+    es6: true,
+    node: true
```

Comment:
I usually don't add this here (prefer to do it per module) but this project is 100% `node`, so whatever.